cleanup: remove unnecessary require for global Buffer

Phillip9587 opened the pull request:
The Buffer object is globally available in Node.js, so there is no need to explicitly require it.

wesleytodd reviewed (approved):
Can we update the history.md file? Sorry for being pedantic, but until we automate it I want to make sure we dont miss something in the release notes on accident.

Phillip9587 commented:
@wesleytodd ~~Does this require an entry in HISTORY.md? The changes don't alter any behavior; they simply remove unnecessary requires of the global buffer.~~

Edit: I just read your comment on #6147 and #6176 and just added the entry

Phillip9587 committed:
cleanup: remove unnecessary require for global Buffer

The Buffer object is globally available in Node.js, so there is no need to explicitly require it.

wesleytodd commented:
Thanks! yeah we will be automating that, but I think there is some question still of how deep we want to go on the automation, so it has not happened yet. Just making sure we don't make a mistake by abandoning the process even if it is a bad process 😄 .
